sdk/identity/identity/src/client/cliCredentialClient.ts

```diff
@@ -0,0 +1,15 @@
+import * as child_process from "child_process";
+
+export class CliCredentialClient {
```

Comment:
I would prefer not to name this a client. I also think making this a newable class is a little overkill. I think as designed api-extractor will insist on exporting the client class as well since CliCredential is exported and CliCredentialClient is needed to construct it. If on the other hand CliCredential just took a callback to run a command, we'd have fewer types and a simpler interface. Thoughts?

Reply:
This cliCredentialClient is similar to IdentityClient. All existed credentials get access token by using the IdentityClient. To follow this pattern, CliCredential should use a client. There are two possible options for CliCredential to get access token. Option#1 is adding a method createProcess to class IdentityClient， CliCredential call method createProcess in getToken method. Option#2 is putting method createProcess in a new client. We are using Option#2 now.  Which option do you think is better here?

Reply:
Maybe we can rename the cliCredentialClient to cliClient?

Reply:
RE: client name - 
Since this PR is part of a larger effort to implement CLI creds, let's be consistent across SDK implementations for now and track this as an open question to be had in arch review in the new year.  Thank you.